chord_metadata_service/restapi/api_views.py

```diff
@@ -355,3 +357,21 @@ def public_overview(_request):
         }
 
     return Response(response)
+
+
+@api_view(["GET"])
+@permission_classes([AllowAny])
+def get_stats_summmary(_request):
```

Comment:
1) "summary" has too many m's in it, but 2) this should probably be named something more like `moh_overview` instead, so that it matches the other names, and probably will eventually be the overview of the MoH model packets.

Reply:
fixed